.husky/pre-commit

```diff
@@ -1,21 +1,7 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
 
-# Redirect output to stderr.
-exec 1>&2
- 
-# prevent it.only or describe.only commited
-if [ "$allowonlytests" != "true" ] &&
-    test $(git diff --cached | grep -E "\b(it|describe).only\("  | wc -l) != 0
-then
-    cat <<\EOF
-Error: Attempt to add it.only or describe.only - which may disable all other tests
- 
-If you know what you are doing you can disable this check using:
- 
-    git config hooks.allowonlytests true
-EOF
-    exit 1
-fi
```

Comment:
why remove the check for `it.only` in tests?

Reply:
I thought this was a placeholder and not used. I made `husky` regenerate the file from scratch. Do you think this is relevant?

Reply:
Well, it prevented me from accidentally pushing `only` to a PR. I use it quite often to debug a specific test and avoid running other ones.

Reply:
I understand, I added the check back